crates/language_tools/src/lsp_log.rs

```diff
@@ -540,7 +540,6 @@ impl LogStore {
             IoKind::StdOut => true,
             IoKind::StdIn => false,
             IoKind::StdErr => {
-                let message = format!("stderr: {}", message.trim());
```

Comment:
I agree with removing this, having `stderr: ` on every output line isn't helpful unless we're interleaving it with other output and you need to be able to differentiate. @SomeoneToIgnore it looks like you were the one that added this, do you know if there's any other output here?

Reply:
IIRC, stdout is not available to use at all when working with langservers?

So, no good reason to keep it here, was added during moving another code around and if the above is true, it's pretty useless.